Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apt-get update && apt-get install -y qemu-system-x86 rsync qemu-kvm minicom
     rm -rf /var/lib/apt/lists/* && \
     pip install --upgrade pip && \
     pip install robotframework==3.0 requests==2.4.3 robotframework-requests==0.4.5 pylibftdi==0.15.0 && \
-    npm install --global resin-cli@7.9.0 && \
+    npm install --global resin-cli@7.10.6 && \
```

Comment:
Maybe after we release balena-cli we could also change this to be the latest prod version of the cli instead of a pinned one.